Adding startup_lite alternative startup script

kellrott opened the pull request.

bgruening commented:
Thanks @kellrott. Is there any reason you want to include your own job_conf and not copying `/galaxy-central/config/job_conf.xml.sample_basic`

kellrott commented:
I've changed it so it uses the job_conf.xml.sample_basic by default. I also added a '-j' flag that means it won't try to override the GALAXY_CONFIG_JOB_CONFIG_FILE environmental variable.

bgruening commented:
@kellrott this is awesome! Less changes but a new powerful mode. Thanks!
